src/icon.h

```diff
@@ -472,3 +495,30 @@ static const unsigned char  Gaggia_Classic_Logo[] = {
     0x00, 0x60, 0x00, 0x00, 0x00, 0x20, 0x00, 0x20, 0x00, 0x00, 0x00, 0x20,
     0x00, 0xe0, 0xff, 0xff, 0xff, 0x3f
 };
+static const unsigned char  Gaggia_Classic_Logo_new[] = {
```

Comment:
also now "new" in vars

Reply:
Hi, i thought the Logos where already merged to the Master?
I have used Heiz, because there was Heiz used in the old Code.

Reply:
Hey, no this is not merged, the pr would be closed then.

Yes, "Heiz" was already used, still not nice.
So I would change that in your pr and later on in the whole code.
But feel free to change that globaly.